chromium_src/chrome/browser/ui/browser_navigator.cc

```diff
@@ -0,0 +1,10 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/browser/renderer_host/brave_navigation_ui_data.h"
+#include "chrome/browser/renderer_host/chrome_navigation_ui_data.h"
```

Comment:
this line is redundant

Reply:
Addressed in https://github.com/brave/brave-core/pull/583/commits/2a7f3440fb5911a296474ff02e3930cb35977693, thanks.